lib/util/exec/index.ts

```diff
@@ -61,14 +62,15 @@ function createChildEnv(
   });
   const extraEnvKeys = Object.keys(extraEnvEntries);
 
-  const childEnv =
+  let childEnv =
     env || extraEnv
       ? {
           ...extraEnv,
           ...getChildProcessEnv(extraEnvKeys),
           ...env,
         }
       : getChildProcessEnv();
+  childEnv = { ...childEnv, ...execConfig.customEnvForChild };
```

Comment:
I think this throws for null or undefined `customEnvForChild`

Reply:
Hmm, the following didn't fail on Node 12. :thinking: Shall I add a safeguard?
```
Welcome to Node.js v12.19.0.
Type ".help" for more information.
> let obj = { "k": "v" }
undefined
> {...obj, ...null}
{ k: 'v' }
> {...obj, ...undefined}
{ k: 'v' }
```

Reply:
No, maybe it failed on an older version. Leave it as is.